core/src/main/java/org/elasticsearch/action/admin/indices/warmer/delete/TransportDeleteWarmerAction.java

```diff
@@ -92,6 +92,7 @@ public ClusterState execute(ClusterState currentState) {
                 MetaData.Builder mdBuilder = MetaData.builder(currentState.metaData());
 
                 boolean globalFoundAtLeastOne = false;
+                boolean deleteAll = Arrays.asList(request.names()).contains("_all");
```

Comment:
can we use the `MetaData.ALL` constant here? and maybe just iterate over the array instead of creating a list just for that?